domain/application/state/application.go

```diff
@@ -1436,6 +1442,28 @@ WHERE name = $minimalUnit.name
 	return unit.LifeID, nil
 }
 
+func (st *State) getMinimalUnit(ctx context.Context, tx *sqlair.TX, unitUUID coreunit.UUID) (minimalUnit, error) {
```

Comment:
`minimalUnit` is not a nice name. What if someone else decides that a minimal unit is more than a UUID, NetNodeID, Name, and LifeID, is it still minimal then?

Reply:
renamed method